app/assets/javascripts/models/beta/story.js

```diff
@@ -493,3 +506,41 @@ export const donePoints = (stories) =>
 
 export const remainingPoints = (stories) =>
   totalPoints(stories) - donePoints(stories);
+
+export const normalizeStories = (data) => {
+  const normalizedState = {
+    stories: {},
+  };
+
+  data.forEach((story) => {
+    const storyId = story.id;
+
+    normalizedState.stories[storyId] = {
+      ...story,
+    };
+  });
+
```

Comment:
Can we use map and spread here instead to avoid mutability? Also, the `{ ...story }` is not needed here, if we always consider our objects immutable we don't need to create copies of them if we won't add anything else to the copy.

Reply:
I updated this function using `reduce`!